packages/types/src/Method.ts

```diff
@@ -147,19 +147,20 @@ export default class Method extends Struct {
   }
 
   get args (): Array<Base> {
-    return (this.getAtIndex(1) as Struct<Base>).values();
+    // FIXME This should return a Struct instead of an Array
+    return [...(this.get('args') as Struct).values()];
```

Comment:
@jacogr I think in the UI you use `method.args()` as an array. So Map or Array?

Reply:
Prefer array. Plus it is actually closer to `.values()`